bndtools.m2e/src/bndtools/m2e/MavenRunListenerHelper.java

```diff
@@ -152,6 +163,21 @@ default MojoExecution getMojoExecution(IMavenProjectFacade projectFacade, String
 			.orElse(null);
 	}
 
+	default String createErrorMessage(String pluginId) {
+		StringBuilder builder = new StringBuilder("Maven Plugin ");
+		builder.append(pluginId);
+		builder.append(" could not be found in your projects maven setup. Please make sure you configured ");
+		builder.append(pluginId);
+		builder.append(" for the bndrun file you want to resolve. See ");
+		if ("bnd-testing-maven-plugin".equals(pluginId)) {
+			builder.append("https://github.com/bndtools/bnd/tree/master/maven/bnd-testing-maven-plugin");
```

Comment:
We must not have such urls in the source code. We don't want to have to change this code should we change things like repo names or branch names.

Reply:
I see your point, but this is our official documentation for the maven plugins. Without it, the whole message would become rather long and unreadable, as I can't use any formating in the Staus.
I can build a Dynamic link, that uses our current version and points to a tag in the repo. Would this be a possible solution?

Reply:
I don't think that is still a good idea. It would be better to link to bnd.bndtools.org since we can control that url namespace. But better still is to put in enough information so the reader knows what to do to fix the issue.

Reply:
Even links to bnd.bndtools.org will become wrong as Bnd changes over time and the old code points to the latest docs. That is an issue with links into the git repo too.

Reply:
To my knowledge, the only documentation for the maven plugins is availalbe as readme.md on github.

Reply:
Okay, I removed the link and changed it to RTFM.